osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -1493,7 +1503,10 @@ def os_clients(all_hosts, all_client_options):
             shutdown_asyncgens_end = time.perf_counter()
             self.logger.info("Total time to shutdown asyncgens: %f seconds.", (shutdown_asyncgens_end - run_end))
             for s in opensearch.values():
-                await s.transport.close()
+                if asyncio.iscoroutinefunction(s.transport.close):
+                    await s.transport.close()
+                else:
+                    s.transport.close()
```

Comment:
Ideally there should be a sync and an async worker coordinator that have a `close` method, or a `transport` class that wraps the async version so that you can change this code to just `s.transport.close()`.

Reply:
This could be refactored in a future check-in.